recipe-ts-runtime/src/Cake.ts

```diff
@@ -121,4 +121,14 @@ export class Cake {
     public getNamespace(): string {
         return this.prefixStack.join(Cake.SEPARATOR);
     }
+
+    public toJson(): string {
```

Comment:
This should probably be `toJSON` so it overrides the `Object.prototype` version. Then if you do `JSON.stringify(aCake)` it will invoke this...

Reply:
I think it can also just return `this.entries`, and `JSON.stringify` of this object will do the work to (eventually) convert to the string version.

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify